Automattic/wp-calypso
# Remove image feature for image thumbnail

#### Changes proposed in this Pull Request

- Adds a remove button to the image so that the in memory information of the image as well as the UI thumbnail can be removed.

![removeimage](https://user-images.githubusercontent.com/3422709/156312328-0016ffa3-c4d4-4492-99cc-70198e86e71a.gif)

#### Testing instructions
* UI feature test
     * Complete a DIFM flow
     * Go to the website content step
     * Try adding any image in the logo upload step or any subsequent step
     * Try removing any of the above images


* Make sure all unit tests are running



Related to #
